emission/export/export.py

```diff
@@ -0,0 +1,78 @@
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import division
+from __future__ import absolute_import
+# Exports all data for the particular user for the particular day
+# Used for debugging issues with trip and section generation
+from future import standard_library
+standard_library.install_aliases()
+from builtins import *
+import sys
+import logging
+logging.basicConfig(level=logging.DEBUG)
+import gzip
+
+import uuid
+import datetime as pydt
+import json
+import bson.json_util as bju
+import arrow
+import argparse
+
+import emission.core.wrapper.user as ecwu
+import emission.storage.timeseries.abstract_timeseries as esta
+import emission.storage.timeseries.timequery as estt
+import emission.storage.decorations.user_queries as esdu
+import emission.storage.timeseries.cache_series as estcs
+# only needed to read the motion_activity
+# https://github.com/e-mission/e-mission-docs/issues/356#issuecomment-520630934
+import emission.net.usercache.abstract_usercache as enua
+
+
+def export(loc_time_query, trip_time_query, place_time_query, ma_entry_list, user_id, file_name):
```

Comment:
I am not sure you need to have duplicated queries here. Are the duplicated queries actually different? Or can you simplify this significantly?

Reply:
I will need to thin these out and plan to do so.